x/concentrated-liquidity/msg_server.go

```diff
@@ -64,7 +64,7 @@ func (server msgServer) WithdrawPosition(goCtx context.Context, msg *types.MsgWi
 		return nil, err
 	}
 
-	amount0, amount1, err := server.keeper.withdrawPosition(ctx, msg.PoolId, sender, msg.LowerTick, msg.UpperTick, msg.LiquidityAmount)
+	amount0, amount1, err := server.keeper.withdrawPosition(ctx, msg.PoolId, sender, msg.LowerTick, msg.UpperTick, msg.LiquidityAmount.ToDec())
```

Comment:
why is the message not giving a dec in the first place?

Reply:
We need to revisit the choice of `Dec` vs `Int` for storing liquidity. I'm yet convinced that `Dec` is what we want